lib/suse/connect/api.rb

```diff
@@ -205,12 +205,21 @@ def system_activations(auth)
       # @param auth [String] authorization string which will be injected in 'Authorization' header in request.
       #   In this case we expect Base64 encoded string with login and password
       # @param [Array <Remote::Product>] a list of producs
+      # @param target_base_product [Remote::Product] (optional) a target base
```

Comment:
Needs to specify that this is only used in the :offline case

Reply:
Added!